<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EMU8000 Control &amp; Debug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="mixvol_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Mixer Volume</string>
    </property>
    <widget class="QGroupBox" name="mmaster_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Master</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="master_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>23</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="msynth_box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Synth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="synth_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>31</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="mixeq_box">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Mixer EQ</string>
    </property>
    <widget class="QGroupBox" name="mbass_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Bass</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="mbass_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="mtreble_box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Treble</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="mtreble_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="syntheq_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Synth EQ</string>
    </property>
    <widget class="QGroupBox" name="sbass_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Bass</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="sbass_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="streble_box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Treble</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSlider" name="streble_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="reverb_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Reverb</string>
    </property>
    <widget class="QComboBox" name="reverb_cmbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Room 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Room 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Room 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hall 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hall 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Panning Delay</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="chorus_box">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Chorus</string>
    </property>
    <widget class="QComboBox" name="chorus_cmbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Chorus 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chorus 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chorus 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chorus 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Feedback</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flanger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Short Delay 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Short Delay 2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="misc_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Misc</string>
    </property>
    <widget class="QCheckBox" name="s3d_chbox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Sound</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="chset_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Channel Settings</string>
    </property>
    <widget class="QGroupBox" name="ch_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>CH: 1</string>
     </property>
     <widget class="QDial" name="ch_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>1.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cc_box">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>CC: 0</string>
     </property>
     <widget class="QDial" name="cc_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>7.937500000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="vol_box">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Vol: 0</string>
     </property>
     <widget class="QDial" name="vol_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>7.937500000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ccval_box">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Val: 0</string>
     </property>
     <widget class="QDial" name="ccval_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>7.937500000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pan_box">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Pan: 0</string>
     </property>
     <widget class="QDial" name="pan_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-64</number>
      </property>
      <property name="maximum">
       <number>63</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>7.937500000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pgm_box">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Pgm: 0</string>
     </property>
     <widget class="QDial" name="pgm_dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>7.937500000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connection_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rstset_btn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
